.travis.yml

```diff
@@ -1,66 +1,39 @@
+# Config file for automatic testing at travis-ci.org
+
 language: python
 
 # Use container based infrastructure
 sudo: false
 
-python:
-  - 2.6
-  - 2.7
-  - 3.3
-  - 3.4
-
 env:
   global:
     - secure: lNbb7g6bjHssNvj02PMAhdRMLTje0+WsYRxy7hdoPGJd5uCm3rx5Ha7yYmpv+6hj8SIA6QuE0viKVnife2Eyets99mMuYtsxE+NfHr19HpPEQPUdph3C5q93tupv+vJ52NQnPkb92PNAxDcM4+F+NQrlUBzVhoPjD/8y3whjFmE=
     - secure: PiQisj0TKzfcoSuJe8nCiE3oKz1OT8IW+9lLA73lrHutY/yJrDlGrLoFBp20iIquVOKpGPX1DH1GvHVZeoiiV3qpaF/lJXmmacJte/FrdiVtQ9ezQYtRkY0UGiF+WnqfBPyGBHma9LszGcKHUIg+sXyQ/tIFFhq7U1vMKSPTHJ4=
     - secure: gCi1UP9rgOnfvLT+UNiQkysnXUKvMi5ichNwPNxiXkLYlc66bJLIc/h8YI3V9vV4ICzyMJqEDHXV7T8NP0tsdxdNdEQNvStoaleLz7xsbjW7MZ2XxF13xncG6fksYr/5cjhBbiwg06HTn/VH+TD1FT1u9WUe+0/BJSx1JMnJQgs=
```

Comment:
Can someone add a comment in the config for each of these lines as to specifically what they're for. Otherwise, they'll be here forever, as no one will every feel like they can remove them.